spec/features/user_edit_profile_picture_spec.rb

```diff
@@ -0,0 +1,45 @@
+require 'rails_helper'
+
+RSpec.describe 'Profile picture', type: :feature do
+
+  let(:user) { FactoryBot.create :user }
+  let(:contact) { FactoryBot.create :user }
+
+  it 'is a default profile-picture shown on the user-show-page, when no individual picture was set' do
+    sign_in user
+    visit user_path(user)
+    expect(page).to have_css("img[src*='avatar-default-test.png']")
+  end
+
+  it 'is possible to upload a profile picture' do
+    sign_in user
+    visit edit_user_path(user)
+    attach_file 'user_avatar', 'spec/support/assets/new-avatar.jpeg'
+    click_button 'Update User'
+    visit user_path(user)
+    expect(page).to have_css("img[src*='new-avatar.jpeg']")
+  end
+
+  it 'is not possible to upload a picture larger then 5 MB' do
+    sign_in user
+    visit edit_user_path(user)
+    attach_file 'user_avatar', 'spec/support/assets/avatar-big.jpeg'
+    click_button 'Update User'
+    expect(page).to have_text('Avatar File size should be less than 5 MB')
```

Comment:
can we ask something like: have_error? Because testing on the string is quite brittle

Reply:
and where is this string defined 🙈

Reply:
This is the standard error message for this.
I think that is why it is okay to test the string, because it isn't a customized one.

Reply:
ah alright yep than I think its fine!

Reply:
[This](https://guides.rubyonrails.org/i18n.html#error-message-scopes) would be the way to properly internationalize standard error messages. The defined translation could then also be used in tests. We did not do this anywhere so this can be deferred in this PR as well